armi/cases/case.py

```diff
@@ -769,10 +769,9 @@ def copyInterfaceInputs(
         # guess. In future, it might be nice to have interfaces specify which
         # explicitly.
         for key, files in interfaceFileNames.items():
-            if isinstance(key, settings.Setting):
-                label = key.name
-            else:
-                label = key
+            if not isinstance(key, settings.Setting):
+                key = cs.getSetting(key)
```

Comment:
It might make sense to capture this in a try/except and provide a better exception that will point us to the specifyInputs method(s) providing an incorrect setting name.